src/entities/priceImpact/addLiquidityNested.ts

```diff
@@ -134,6 +134,7 @@ async function getAddBoostedUnbalancedResult(
         rpcUrl,
         amountsIn,
         kind: AddLiquidityKind.Unbalanced,
+        wrapUnderlying: [true, true], // TODO: Handle more elegantly?
```

Comment:
Yeah, should probably handle correctly if we are going to support this. Can you figure out an example of what this looks like and if wrap underlying is valid - I think its a nested boosted but haven't thought it through properly.

Reply:
I think this may be a decent way to handle  ( until composite router v3 with support for both boosted and nested )

https://github.com/balancer/b-sdk/blob/12d5255f94dce0268ce9b18b3be1ea836ec28117/src/entities/priceImpact/addLiquidityNested.ts#L67

Reply:
turns out this was just a symptom of bad initial implementation plan :sweat: